src/utils/RAML.js

```diff
@@ -184,6 +184,9 @@ export default class RAML {
                             if (methodClassFunction === 'fetch') {
                                 debug(`Special case, this is a POST or PATCH at {id}, which is an update()`);
                                 methodClassFunction = 'update';
+                            } else if (methodClassFunction === 'list' && method.method === 'post') {
```

Comment:
Hi, it was what I did first but the switch statement catch post AND patch. It would be weird if the PATCH could match with the create function.

I'm not fond of this condition but that is the easiest solution I found without a bigger refactoring.